tests/phpunit/Unit/Serializers/QueryResultSerializerTest.php

```diff
@@ -237,7 +237,9 @@ private function buildMockQueryResult( $setup ) {
 			] );
 
 			$printRequests[] = $printRequest;
-			$getResults[] = \SMW\DIWikiPage::newFromTitle( new \Title( NS_MAIN, $value['printRequest'] ) );
+			# TODO: Should be the following line, but the environment of the test should be adapted
+			# $getResults[] = \SMW\DIWikiPage::newFromTitle( \Title::makeTitle( NS_MAIN, $value['printRequest'] ) );
+			$getResults[] = \SMW\DIWikiPage::newFromTitle( \Title::makeTitle( NS_MAIN, '' ) );
```

Comment:
This is an issue in the test: it is tested against an empty title. Previously it was created a Title object with `new Title()`, the constructor of Title is now private, so I used `Title::makeTitle( NS_MAIN, '' )` to continue creating it with the same behaviour, but it is a quickfix and the test should instead be adapted.